pkg/cli/config.go

```diff
@@ -95,20 +95,44 @@ func BaseConfig() *Config {
 	}
 }
 
-func LoadConfig(srcs []string, c *Config) error {
-	combined := []byte{}
-	for _, path := range srcs {
-		b, err := os.ReadFile(path)
+func LoadConfig(sysSrc, userSrc string, c *Config) error {
+	loadFn := func(src string, out *Config) error {
```

Comment:
nit: I'd prefer this function to be outside `LoadConfig`. Maybe renaming `LoadConfig` as `LoadConfigs` and naming this one `LoadConfig` or `ExtendConfigFromFile`

Reply:
Done, moved it to package private function: `func loadConfig(src string, out *Config) error`.